src/test/java/org/example/c20debugging/BankAccountTestParameterized.java

```diff
@@ -0,0 +1,49 @@
+package org.example.c20debugging;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import static junit.framework.TestCase.assertEquals;
+
+@RunWith(Parameterized.class)
+public class BankAccountTestParameterized {
+
+    private BankAccount account;
+    private double amount;
+    private boolean branch;
+    private double expected;
+
+    public BankAccountTestParameterized(double amount, boolean branch, double expected) {
+        this.amount = amount;
+        this.branch = branch;
+        this.expected = expected;
+    }
+
+    @org.junit.Before
```

Comment:
Use `org.junit.jupiter` imports instead of `org.junit`

Reply:
done